CHANGES.rst

```diff
@@ -40,6 +40,9 @@ astropy.coordinates
   by default. This currently corresponds to the values defined in v4.0, but will
   change with future releases. [#10238]
 
+- The ``SkyCoord.from_name()`` and Sesame name resolving functionality now
+  optionally caches results locally. [#9162]
```

Comment:
Darn - should remove "optionally" since it is the default. Maybe "now is able to cache results locally and will do so by default"

Reply:
Good point! Thanks for catching this